### PR TITLE
Update New-NetQosPolicy.md

### PR DESCRIPTION
**Pull request by sowjanya365**
Issue #1874 - Bits and Bytes conversion

### CODE REVIEW

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/netqos/New-NetQosPolicy.md [hunk 1]
```diff
@@ -162,7 +162,7 @@ AppPathName    : ftp.exe
 ThrottleRate   : 1 MBytes/sec
 ```
 
-This command creates a QoS policy named FTP that matches an application path at ftp.exe and throttles the traffic at 1,000,000 bits per second.
+This command creates a QoS policy named FTP that matches an application path at ftp.exe and throttles the traffic at 8,388,608 bits per second.
```
Comment (dariomws) [inline docset/windows/netqos/New-NetQosPolicy.md:165]:
```suggestion
This command creates a QoS policy named FTP that matches an application path at ftp.exe and throttles the traffic at 8,000,000 bits per second.
```

File: docset/windows/netqos/New-NetQosPolicy.md [hunk 2]
```diff
@@ -193,7 +193,7 @@ IPPort         : 80
 ThrottleRate   : 10 MBytes/sec
 ```
 
-This command creates a QoS policy that matches TCP traffic sent to port 80 and rate-limits it at 10,000,000 bits per second.
+This command creates a QoS policy that matches TCP traffic sent to port 80 and rate-limits it at 83,886,080 bits per second.
```
Comment (dariomws) [inline docset/windows/netqos/New-NetQosPolicy.md:196]:
```suggestion
This command creates a QoS policy that matches TCP traffic sent to port 80 and rate-limits it at 80,000,000 bits per second.
```

File: docset/windows/netqos/New-NetQosPolicy.md [hunk 3]
```diff
@@ -220,7 +220,7 @@ URIRecursive   : False
 ThrottleRate   : 500 KBytes/sec
 ```
 
-This command creates a QoS policy named IIS that matches return traffic from an HTTP server application with the specified URI and rate limit the return traffic at 500,000 bits per second.
+This command creates a QoS policy named IIS that matches return traffic from an HTTP server application with the specified URI and rate limit the return traffic at 4,096,000 bits per second.
```
Comment (dariomws) [inline docset/windows/netqos/New-NetQosPolicy.md:223]:
```suggestion
This command creates a QoS policy named IIS that matches return traffic from an HTTP server application with the specified URI and rate limit the return traffic at 4,000,000 bits per second.
```

**Comment by sowjanya365** [comment]
Thank you, @dariomws! :)

**Pull request merged**